app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -657,6 +657,10 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
                     return true;
                 }
                 break;
+            case KeyEvent.KEYCODE_B:
+            case KeyEvent.KEYCODE_T:
+                EventBus.getDefault().post(event);
```

Comment:
You can just post all KeyEvents, not just B and T. Makes it easier to add more in the future.

Reply:
Hmm, I don't really want to merge something that is only half finished. Would be great if you could move sending the event up, so it is the first thing the method does, independently of the key code.